contents/blog/multi-product-pricing.md

```diff
@@ -0,0 +1,73 @@
+
+The moment we started charging, more users realized the feature existed, the more seriously they took it, and the more demanding their feedback. Paying users are less bug tolerant!
+
+We put a small team on it, who quickly iterated with customers and improved the quality.
+
+Growth of session replay took off - it now has similar daily usage to product analytics.
+
+## Focusing on paying customers can improve the free product
+
+Intuitively, if someone is paying, they will have a higher expectation.
+
+However, the secondary effect we didn't anticipate is that fixing the smaller issues can lead to much more usage from both paid and free users. It turns out that in 2021, a lot of UX weirdness and buginess we had was affecting all users of our platform, but our free users weren't complaining that loudly.
+
+As context, PostHog spent the first year and a half prioritizing our open source project - we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product - product analytics.
+
+Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience - a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.
+
+This journey has since repeated itself with session recording. We have a small team of 2 working on this product, and we set them the goal in Q4 last year of charging for their product and getting their first paying customers. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements. 
+
+And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.
+
+## Our current thining - outcompeting on price, sustainably
+
+Do you want to buy 5 individual products at $10 each, or all 5 for $20?
+
+Charging by product means you can outcompete all of your competitors on price.
+
+To get to this point, we're starting to build data on cross-selling to know that we'd get it right, so we'll figure that out over the next year and hopefully will be able to lower prices as a result.
+
+If you're thinking about applying this concept to your business, success depends on where in the market you focus (you need to be able to sell more to the same users, ideally - to avoid a complex sales motion) and how price sensitive they are. If people don't care about price in your market, there's no point either.
```

Comment:
```suggestion
## 4. It allows us to compete on price sustainably

Do you want to buy 5 individual products at $10 each, or all 5 for $20?

Charging by product means you can outcompete all of your competitors on price.

To get to this point, we're starting to build data on cross-selling to know that we'd get it right, so we'll figure that out over the next year and hopefully will be able to lower prices as a result.

If you're thinking about applying this concept to your business, success depends on where in the market you focus (you need to be able to sell more to the same users to avoid a complex sales motion) and how price sensitive they are. If people don't care about price in your market, there's no point either.
```